samples/sp_whoIsActive/src/controllers/mainController.ts

```diff
@@ -49,15 +49,9 @@ export default class MainController extends ControllerBase {
     }
 
     private onExecute(connection: sqlops.IConnectionProfile, fileName: string): void {
-        let sqlFile = fs.readFileSync(path.join(__dirname, '..', 'sql', fileName)).toString();
-        this.openSQLFileWithContent(sqlFile);
-    }
-
-    private openSQLFileWithContent(content: string): void {
-        vscode.workspace.openTextDocument({language: 'sql', content: content}).then(doc => {
-            vscode.window.showTextDocument(doc, vscode.ViewColumn.Active, false);
+        let sqlContent = fs.readFileSync(path.join(__dirname, '..', 'sql', fileName)).toString();
```

Comment:
Why not add an API call to open a SQL editor from an existing file? It will probably be a pretty common use case

Reply:
You shouldn't need this at all. The existing vscode workspace API `export function openTextDocument(options?: { language?: string; content?: string; }): Thenable<TextDocument>` will work. It opens an untitled document with the specified language (sql) and contents (the text). Have you tried using this?

Reply:
Basically any file open / create new untitled file code should "just work", and if it doesn't it means our handler for SQL language type isn't working as expected